pub/lib/dependabot/pub/update_checker.rb

```diff
@@ -61,11 +46,41 @@ def updated_requirements
 
       private
 
+      # Returns unparsed_version if it looks like a git-revision.
+      #
+      # Otherwise it will be parsed with Dependabot::Pub::Version.new and
+      # checked against the ignored_requirements:
+      #
+      # * If not ignored the parsed Version object will be returned.
+      # * If allowed_version is non-nil and the parsed version is the same it
```

Comment:
What would you think of calling this `current_version` instead of `allowed_version`? Elsewhere we tend to use `allowed` to refer to versions that it's possible to update to but if I followed this code correctly this would be the version that is currently in the `pubspec.lock`?

Reply:
Renamed